cache/cache.cpp

```diff
@@ -190,11 +194,332 @@ size_t cache_manager::idx(const unsigned char *id) {
 	return (i ^ j) % m_caches.size();
 }
 
+write_request::write_request(struct dnet_io_attr *io, char *data)
```

Comment:
Please, keep the order of `write_request` and `cache_manager` the same in header and source files.